test/e2e/vsphere_test.go

```diff
@@ -373,6 +373,46 @@ func TestVSphereKubernetes121BottleRocketWorkloadClusterCuratedPackagesSimpleFlo
 	runCuratedPackageRemoteClusterInstallSimpleFlow(test)
 }
 
+func TestVSphereKubernetes121BottleRocketWorkloadClusterCuratedPackagesCertManagerSimpleFlow(t *testing.T) {
```

Comment:
Are you going to add Ubuntu tests in a separate PR? If not, try to do a mix of BottleRocket and Ubuntu tests here for better coverage.

Reply:
I think a single OS and a single provider should suffice Cert manager since cert manager doesn't have any dependency on OS or provider.